eslint.config.js

```diff
@@ -1,14 +1,21 @@
+import eslintReact from "@eslint-react/eslint-plugin";
```

Comment:
I think your config is almost correct, you can ignore also the graphql generated files but try to run the commands 
npm run prettier and npm run lint, you will see some differences too, for next time a pre-commit hook configuration would help you